main/http_server/axe-os/src/app/layout/app.layout.component.ts

```diff
@@ -1,6 +1,7 @@
 import { Component, OnDestroy, Renderer2, ViewChild } from '@angular/core';
 import { NavigationEnd, Router } from '@angular/router';
-import { filter, Subscription } from 'rxjs';
+import { filter, Subscription, Subject, takeUntil } from 'rxjs';
+import { SensitiveData } from 'src/app/services/sensitive-data.service';
```

Comment:
Terminology is mixed, `sensitive-content` (css class) and `SensitiveData` (here).